app/views/admin/editions/_change_notes.html.erb

```diff
@@ -1,24 +1,47 @@
 <fieldset class="js-change-notes">
-  <h3 class="remove-top-margin">What sort of change are you making?</h3>
-  
-  <div class="radio">
-    <%= form.label :minor_change_true do %>
-      <%= form.radio_button(:minor_change, true) %>
-      <strong>Fixing a typo or broken link, a style change or similar</strong>
-      <p class="hint">Users signed up to email alerts won't be notified</p>
-    <% end %>
-  </div>
+  <h3 class="remove-top-margin">Do users need to know the content has changed?</h3>
+  <p class="hint">Choose yes to notify users of the change. Telling users when published information changes is important for transparency.</p>
+
+  <% if edition.allows_attachments? && edition.has_attachment_changes? %>
+    <div class="alert alert-info">
+      <h4> Attachments you've added, changed or deleted </h4>
+      <ul class="govuk-list">
+      <% if edition.has_created_attachments? %>
+        <% edition.created_attachments.each do |attachment| %>
+          <li><%= "#{attachment.title} - #{attachment.readable_type} attachment" %> <span class="label label-success">Created</span></li>
+        <% end %>
+      <% end %>
+      <% if edition.has_updated_attachments? %>
+        <% edition.updated_attachments.each do |attachment| %>
+          <li><%= "#{attachment.title} - #{attachment.readable_type} attachment" %> <span class="label label-warning">Updated</span></li>
+        <% end %>
+      <% end %>
+
+      <% if edition.has_deleted_attachments? %>
+        <% edition.deleted_attachments.each do |attachment| %>
+          <li><%= "#{attachment.title} - #{attachment.readable_type} attachment" %> <span class="label label-danger">Deleted</span></li>
+        <% end %>
+      <% end %>
```

Comment:
For a more mild DRY-up, we should be able to drop these `if` statements because `#each`ing through an empty array will simply do nothing anyway.

```suggestion
      <% edition.created_attachments.each do |attachment| %>
        <li><%= "#{attachment.title} - #{attachment.readable_type} attachment" %> <span class="label label-success">Created</span></li>
      <% end %>
      <% edition.updated_attachments.each do |attachment| %>
        <li><%= "#{attachment.title} - #{attachment.readable_type} attachment" %> <span class="label label-warning">Updated</span></li>
      <% end %>
      <% edition.deleted_attachments.each do |attachment| %>
        <li><%= "#{attachment.title} - #{attachment.readable_type} attachment" %> <span class="label label-danger">Deleted</span></li>
      <% end %>
```